netbsd/NetBSDProcessList.c

```diff
@@ -49,23 +50,64 @@ static char const *freqSysctls[] = {
    NULL
 };
 
+static void NetBSDProcessList_updateCPUcount(ProcessList* super) {
+   NetBSDProcessList* opl = (NetBSDProcessList*) super;
+   const int nmib[] = { CTL_HW, HW_NCPU };
+   const int mib[] = { CTL_HW, HW_NCPUONLINE };
+   int r;
+   unsigned int value;
+   size_t size;
+   bool change = false;
+
+   size = sizeof(value);
+   r = sysctl(mib, 2, &value, &size, NULL, 0);
```

Comment:
Comment what's queried to ease following along.

Reply:
Since `mib` is now documented in the above lines, should I add more info on the sysctl(3) call?

Reply:
Thought more like renaming those variables to `nmib` -> `mib_ncpu_existing` and `mib` -> `mib_ncpu_online`. That way the `sysctl`(3) automagically become semi-documented. For the actual `sysctl`(3) you could just add a short `// Query number of existing CPUs` vs. `// Query number of active/online CPUs` above the code block involving that step.

Reply:
I agree with you here, I was just following the existing convention when doing the implementation. I shall change this and get back to you.

Reply:
Refactored the sections based on the below suggestion.